translation/en/RedirectedResources/assets/abdata/h/list/14_00/personality_voice_c30_01_00/translation.txt

```diff
@@ -1,517 +1,517 @@
+もぉっ、またそんなにつんつんしてぇ…やらしい指つきなんだから♪=Jeez-- You're poking them like that again... Such lewd fingertips they are ♪
+むぅ……キミ、先っぽつつくの、好きなのかしら？=Muu... Do you really like poking my nipples that much?
+ほら、さっきより硬くなってるでしょ？　何回もつついてみて=Look, aren't they stiffer than before? Try poking them even more
+いい感じに硬くなってるでしょ？　キミがいつもつつくからよ♪=Don't they feel good, all nice and stiff like that? It's because you're always poking them, you know ♪
+ふあぁっ、それっ……結構刺激強いのよっ……んんっ、胸っ、びくびくしちゃうっ……=Fuaa-- That-- That's fairly stimulating... Nnn-- I can feel my chest quivering...
+あっ、また引っ張ってるっ…身体ごと持っていかれそうよぉっ！=Ah-- Pulling on them again... It feels like my whole body will be pulled away!
+あはっ……何回されても引っ張られるの好きぃっ……キミにおもちゃにされてる感じがいいのぉっ！=Aha--... No matter how many times you do it, I love having you pull on them... I love it when you treat me like a toy!
+やっ、だっ……先っぽ引っ張られてるだけなのにっ……アソコまで響いちゃうっ……=No--... Even though you're just pulling it on my nipples... I feel it all the way to my pussy...
+あああ……そのまま、おっぱいごと持ち上げてぇっ……あんっ、痛気持ちいいぃっ！=Aaa... Just like that, raise my tits up--... Ahn-- It feels so good even tho it hurts!
+あんっ、引っ張って引っ張ってっ！　この感触、好きなのっ……ゾクゾクしちゃうっ！=Ahn-- Pull on them, Pull on them! This sensation, I love it--... Ah, it's giving me the chills!
+うぅ……キミ、またそんなところ弄って……もう……ヘンタイさんみたい……=Uu... Playing with that place again... Jeez... You're such a pervert...
+うそ…またお尻弄っちゃうの…ほんとうに好きなんだからぁ=No way... You're gonna fiddle with my ass again?... You really do love it, don't you
+んっ、またアナル……そんな丁寧にいじって……もうっ……=Nn-- Again with my asshole... and so careful too... Jeez...
+あ、あんまりじっくり弄らないでね……は、恥ずかしくて死にそうなんだからぁ……=D-Don't grope around so carefully and thouroughly... I-It's so embarrassing I want to die...
+キミのお尻の穴好きには困ったわね……なーんて、お姉さんも好きだから……んんっ、いっぱいして？=It's quite troubling how much you love my asshole... Just kidding, Onee-san loves having you play with it too... Nnn-- Mess it lots, please?
+尻穴……って下品な響き……尻穴弄られてるって口にするだけで、身体中が悦んじゃう……=Asshole... such a vulgur word... yet just hearing you wanting to play with my asshole is enough to make my body react with joy...
+はぅっ、んっ、そこっ、擦られるとっ……痺れっ、ちゃうっ……ふあっ……=Hau--, Nn--, That place-- It's being rubbed... I feel numb... Fuaa...
+もぉっ……何回擦られても愛液出ちゃうっ……水分不足になっちゃいそぉっ！=Jeez--... The more you rub it, the more the love juice will come out... At this rate it won't be enough!
+あぁんっ、また擦ってるぅっ！　オマンコ、激しく擦るとっ……あっ、ほらぁ、どんどんぬるぬるしちゃうでしょっ……=Aaahn-- You're rubbing it again--! If you keep running my pussy this intensely... Ah-- Look, It's getting wetter and wetter...
+うぅぅ……ぬ、ぬるぬるになっちゃってるわよね……自分でも分かるの……恥ずかしい……けど気持ちいい……=Uuuu... It's becoming really wet, isn't it... I can feel it... It's so embarrassing... But it also feels so good...
+その擦り方っ、好きっ……今度自分でもやってみるわね=That technique of yours, I love it... I'll try it myself next time
+あっ、あっ、もぉっ、穴ギリギリのところ指で擦るのっ……あぁんっ、もどかしくて濡れちゃうっ！=Ah-- Ah-- Jeez-- If you keep rubbing just the edge of the asshole like that... Aaahn-- It's going to become wet from impatience!
+ま、待って……お尻、そんなに擦ったらっ、あ、熱くなっちゃうっ……=W
```

Comment:
Did you loose a translation here? Looks like it was in some of your earlier commits.

Reply:
Added missing translation